File: alternatives/server-micronaut-java/src/main/java/blog/Post.java
package blog;

public class Post {
  public String id;
  public String subject;
  public String content;
  public String author;

  public Post() {}

  public Post(String id, String subject, String content, String author) {
    this.id = id;
    this.subject = subject;
    this.content = content;
    this.author = author;
  }

}

File: alternatives/server-django/src/typesadsf/post.py
from pydantic.dataclasses import dataclass
# from .common import DBItem


@dataclass
class DBItem:
    created_at: datetime
    id: UUID
    updated_at: datetime


@dataclass
class Post:
    author: str
    content: str
    subject: str


@dataclass
class DBPost(DBItem, Post):
    pass
